source/Ship.cpp

```diff
@@ -2364,6 +2364,9 @@ int Ship::Scan()
 	// Play the scanning sound if the actor or the target is the player's ship.
 	if(isYours || (target->isYours && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
+	// Potential relation penalties for scanning.
+	if(isYours && target->GetGovernment() && (result & ShipEvent::SCAN_OUTFITS || result & ShipEvent::SCAN_CARGO))
+		target->GetGovernment()->Offend(ShipEvent::SCAN_OUTFITS, target->CrewValue());
```

Comment:
This code isn't needed since reputation hits due to events are handled elsewhere (in `AI::UpdateEvents`).